### PR TITLE
Combat Patrol: Loadouts part 6 (probably)

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
More loadout changes, mainly for SOM this time. Also a few minor typo fixes etc.

**SOM changes:**
V-21 and V-31 loadouts for veterans and SL's. They have access to AP ammunition which gives them higher dps, although it means their damage falls off more sharply (especially the smg). 
Although a caliver should still outdamage them due to deflagrate, against very heavily armoured opponents, the AP ammo is more reliable.

Breacher loadouts for standards and veterans:
The SOM get shields off their own, standards carrying a V-21 (making fuller auto impractical outside of pb range), while the vet gets a volkite charger with gyro, and the new 'Lorica' armour module, the SOM equivilent to Tyr.
image

The volkite charger has been underperforming a bit after several tweaks and nerfs. Slight buffs to wielded accuracy, unwielded scatter, and movement penalties to make it better in the roles it's designed for.

Veterans now use V-11's instead of high power pistols, since they're SOM specific, actually good, and the high power is pretty meh.
**Marine changes:**
Added a flamethrower loadout for standard marines. I'm not certain how useful it will be, but it'll likely be FUN.

The shocktrooper loadout has changed from SH-15 to ALF-51B. The ALF has outstanding damage and mobility, but has basically the worst fall off in the game and lacks underbarrel weapons, meaning you're forced to be very aggressive to get the most out of it, which lines up with what I wanted the shocktrooper loadout to do.

The SH-15 is better for corner peaking due to it's high single shot damage but low ROF. I'll probs add it back in a more suitable loadout later.

## Changelog
:cl:
add: Combat Patrol: Added some more SOM loadouts and gear
add: Combat Patrol: Added a flamethrower loadout to marines
balance: Slight buffs to the volkite charger as it was under performing relative to the caliver
balance: Combat Patrol: Shocktrooper loadout now uses the ALF-51B
fix: Couple of minor typo fixes
/:cl:

### CODE REVIEW

**Comment by XSlayer300** [comment]
Assault Marine does not have mesons in their loadout, intentional?

**Comment by Lumipharon** [comment]
> Assault Marine does not have mesons in their loadout, intentional?

Why would they? Only engies (and vets) have them.

**Review by Kuro020 — APPROVED** [review]

File: code/modules/clothing/modular_armor/attachments/modules.dm [hunk 1]
```diff
@@ -93,16 +93,38 @@
...
 	slot = ATTACHMENT_SLOT_HEAD_MODULE
 	variants_by_parent_type = list(/obj/item/clothing/head/modular/marine/m10x = "tyr_head_xn", /obj/item/clothing/head/modular/marine/m10x/leader = "tyr_head_xn")
 
+/obj/item/armor_module/module/tyr_head/som
+	name = "Lorica Helmet System"
+	desc = "Designed for mounting on a modular SOM helmet. Substantial additional armor improves protection against all damage."
+	icon = 'icons/mob/modular/modular_armor_modules.dmi'
+	icon_state = "lorica_head"
+	item_state = "lorica_head_a"
+	soft_armor = list("melee" = 10, "bullet" = 10, "laser" = 15, "energy" = 15, "bomb" = 15, "bio" = 5, "rad" = 5, "fire" = 10, "acid" = 5)
```
Comment (TiviPlus) [inline code/modules/clothing/modular_armor/attachments/modules.dm:124]: can you use the armor defines please

**Review by TiviPlus — APPROVED** [review]
Rest lgtm

File: code/modules/clothing/modular_armor/attachments/modules.dm (hunk 1 above)
Comment (Lumipharon) [reply code/modules/clothing/modular_armor/attachments/modules.dm:124]: done.

**Review by TiviPlus — APPROVED** [review]

**Comment by Serxule** [comment]
do marines even get shields? i dont think shields will be very fun to fight

**Comment by Serxule** [comment]
fun fact i learned: flamethrowers completely ignore barricades

**Comment by UltraBigRat** [comment]
> do marines even get shields? i dont think shields will be very fun to fight

They have them and they are very much unfun to fight. its literally  just a massive bullet sponge thats very difficult to kill and shoots you with smg.

**Pull request merged**